### PR TITLE
setting up ISO Date based on local timezone

### PR DESCRIPTION
**Pull request by camliu89**
@jivey @nathanstitt this fix is not related to WRM. Is it still ok to push it to the WRM branch?

### CODE REVIEW

**Comment by jivey** [comment]
@camliu89 `DateTimeInput` is new for WRM, are you saying this should also be applied to the assignment builder on the main branch?

**Comment by camliu89** [comment]
oh nvm. Assignment builder is part of WRM. So this fix should be related to WRM. @jivey @nathanstitt do we want this then for the 5/6 release.

**Comment by nathanstitt** [comment]
@camliu89 good catch. I don't think we need it for the 5/6 release tho, lets just put it in WRM so it can be tested along with that code

File: tutor/src/helpers/dates.js [hunk 1]
```diff
@@ -33,9 +33,13 @@ export function findLatest(...dateThings) {
...
 
-
 export function dateWithUnchangedTime(date, t) {
   const time = moment(t);
   return moment(date)
     .hour(time.hour()).minute(time.minute()).seconds(time.seconds()).millisecond(0);
 }
+
+export function guessISODateWithTimezone(isoDate) {
+  const timezone = moment.tz.guess(true);
```
Comment (Dantemss) [inline tutor/src/helpers/dates.js:43]: I don't think this is quite right... it should read the timezone from the course instead of using the browser's.

Comment (camliu89) [reply tutor/src/helpers/dates.js:43]: I might be missing something @Dantemss . I am guessing the timezone of the user here so it reflects the date and time of the course dates in the user's timezone.

**Pull request closed without merging**